cmd/feed-ingress/main.go

```diff
@@ -102,3 +106,42 @@ func readFile(path string) []byte {
 	}
 	return data
 }
+
+func configureHealthPort(controller ingress.Controller) {
+	http.HandleFunc("/health", checkHealth(controller))
+
+	go func() {
+		log.Error(http.ListenAndServe("localhost:"+strconv.Itoa(healthPort), nil))
```

Comment:
Will this be hit by kubelet? If so localhost won't work as it'll be inside a container